src/app/shared/services/models/project.service.ts

```diff
@@ -135,4 +118,52 @@ export class ProjectService extends JsonFeedService {
       })
     );
   }
+
+  /**(
+   *
+   * @param projectModel
+   */
+  loadReadme(projectModel: ProjectModel): Observable<Object> {
+    const rawContentUrl = projectModel.url.replace('https://github.com', 'https://raw.githubusercontent.com');
+    const projectUrl = rawContentUrl + '/master/README.md';
```

Comment:
Will this fail if the default branch is not `master` but `main`? maybe using the other option as a fallback is the first fails is an option. I don't know if GH has a URL for the default branch whatever the name of the branch is.

Reply:
Luckily, master seems to really mean "default branch" so will automatically switch to main! Doesn't work the other away around though!

Reply:
amazing...